File: Front/SalonManagement-App/src/app/models/Servico.ts
import { Cliente } from './Cliente';
import { ProdutoServico } from './ProdutoServico';
import { Profissional } from './Profissional';

export interface Servico {
  id: number;
  data: string;
  descricao: string;
  valor: number;
  profissionalId: number;
  profissional: Profissional;
  clienteId: number;
  cliente: Cliente;
  produtosServicos: ProdutoServico[];
}

File: Back/src/SalonManagement.Domain/Servico.cs
using System.Collections.Generic;

namespace SalonManagement.Domain
{
    public class Servico
    {
        public int Id { get; set; }
        public string Data { get; set; }
        public string Hora { get; set; }
        public string Descricao { get; set; }
        public decimal Valor { get; set; }
        public int ProfissionalId { get; set; }
        public Profissional Profissional { get; set; }
        public int ClienteId { get; set; }
        public Cliente Cliente { get; set; }
        public IEnumerable<ProdutoServico> ProdutosServicos { get; set; }
    }
}